front/app/containers/Admin/projects/all/new/Projects/Filters/ParticipationMethods.tsx

```diff
@@ -72,22 +72,22 @@ const ParticipationMethods = ({ participationMethods, onChange }: Props) => {
   };
 
   return (
-    <Tooltip
-      content={
-        <Box w="240px">
-          {formatMessage(messages.filterByCurrentPhaseMethod)}
-        </Box>
-      }
-    >
-      <FilterSelector
-        name="participation-methods-select"
+    <Box display="flex" alignItems="center">
+      <MultiSelect
         title={formatMessage(messages.participationMethodLabel)}
-        multipleSelectionAllowed
         selected={participationMethods}
-        values={options}
+        options={options}
         onChange={handleOnChange}
       />
-    </Tooltip>
+      <IconTooltip
```

Comment:
I moved the tooltip to an icon tooltip, because the tooltip around the filter it kept stealing the focus and closing the dropdown